notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -346,26 +346,6 @@
         "! gsutil mb -l $REGION -p $PROJECT_ID $BUCKET_URI"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    df_numeric = (df_numeric - df_numeric.mean()) / df_numeric.std()
</pre><p>Shouldn't this normalization happen after the test is set is split from the training set</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>